containers/containers.go

```diff
@@ -86,7 +86,7 @@ type RuntimeInfo struct {
 
 // Store interacts with the underlying container storage
 type Store interface {
-	Get(ctx context.Context, id string) (Container, error)
+	Get(ctx context.Context, id string, fieldpaths ...string) (Container, error)
```

Comment:
I see fieldpaths, fieldmasks and attrmasks. Is there a difference between the values they are carrying?